### PR TITLE
OCPBUGS-17559_RN: Add known issue to 4.13 RNs for Azure File NFS

### PR DESCRIPTION
**Pull request by lpettyjo**
Version(s): 4.13 only

Issue: https://issues.redhat.com/browse/OCPBUGS-17559

Link to docs preview: https://65719--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-known-issues

QE review:
- [x] QE has approved this change.

Additional information:

PR to add same info to Azure File CSI driver content: https://github.com/openshift/openshift-docs/pull/63854

**PTAL**: @bertinatto, @duanwei33, @gcharot, @jsafrane, @sferich888, @xltian, @mwerner2113

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -2854,6 +2854,10 @@ This affects the achievable latency of such clusters and might prevent proper op
 
 * Currently, when using a persistent volume (PV) that contains a very large number of files, the pod might not start or can take an excessive amount of time to start. For more information, see this link:https://access.redhat.com/solutions/6221251[knowledge base article]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1987112[*BZ1987112*])
 
+* Creating pods with Azure File NFS volumes that are scheduled to the control plane node causes the mount to be denied. (link:https://issues.redhat.com/browse/OCPBUGS-18581[*OCPBUGS-18581*])
++
+To work around this issue: If your control plane nodes are schedulable, and the pods can run on workers nodes, use nodeSelector or Affinity to schedule the pod in worker nodes.
```
Comment (tmalove) [inline release_notes/ocp-4-13-release-notes.adoc:2859]:
```suggestion
To work around this issue: If your control plane nodes are schedulable, and the pods can run on worker nodes, use `nodeSelector` or Affinity to schedule the pod in worker nodes.
```
See https://redhat-documentation.github.io/asciidoc-markup-conventions/

**Comment by gcharot** [comment]
LGTM

**Comment by bertinatto** [comment]
lgtm

**Comment by mwerner2113** [comment]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -1008,6 +1008,12 @@ For more information, see xref:../storage/container_storage_interface/persistent
 ==== Delegate FSGroup to CSI Driver instead of Kubelet is generally available
 This feature allows {product-title} to supply a pod's FSGroup to a Container Storage Interface (CSI) driver when a volume is mounted. Microsoft Azure File CSI driver depends on this feature.
 
+[id="ocp-4-13-storage-azure-file-nfs"]
+==== Azure File supporting NFS is generally available
+{product-title} 4.13 supports Azure File Container Storage Interface (CSI) Driver Operator with Network File System (NFS) as generally available.
+
+For more information, see ???.
```
Comment (sferich888) [inline release_notes/ocp-4-13-release-notes.adoc:1015]: We need to remove the ??? here or provide a link?

**Review by sferich888 — APPROVED** [review]
Short of the missing link (additional context) I think this is fine. /lgtm

**Comment by xltian** [comment]
lgtm

**Pull request merged**